### DISCUSSION TITLE
Default workspace on system boot.

### DISCUSSION THREAD

**Discussion by Fi1osof** [mzur/gnome-shell-wsmatrix#244] [Q&A] [answered]
Hello and thanks for awesome extension!
Question: i have 3*3 workspaces matrix. May i set default workspace number for booting, f.e. 5? From this position i can move for 4 directions, not right and down only. This is more usefull for me.

**Comment by mzur** [comment, marked as answer]
This is unrelated to this extension. Please have a look [here](https://askubuntu.com/a/410423/281685) for a possible solution.

**Reply by Fi1osof** [reply]
@mzur ow, cool hack)) Thanks!
